docs/terminal/basics.md

```diff
@@ -22,6 +22,23 @@ To open the terminal:
 
 > **Note:** Open an external terminal with the `kb(workbench.action.terminal.openNativeConsole)` keyboard shortcut if you prefer to work outside VS Code.
 
+## Opening terminals from the Explorer
+
+You can open new terminals for specific folders from the Explorer via the **Open in Integrated Terminal** context menu command.
+
+![Open in Integrated Terminal command in the Explorer context menu](images/basics/open-in-terminal-command.png)
+
+By default, there is no keyboard shortcut associated with **Open in Integrated Terminal** but you can add your own via the Keyboard Shortcuts editor (`kb(workbench.action.openGlobalKeybindings)`) to add a keybinding to your `keybindings.json`.
+
+The `keybindings.json` example below adds the keyboard shortcut `kbstyle(Ctrl+J T)` for `openInTerminal`.
+
+```json
+{
+  "key": "ctrl+j t",
```

Comment:
This would make ctrl+j a chord, breaking the toggle panel visibility keybinding (and maybe others)

Reply:
How about just Ctrl+T and have a when clause so it doesn't conflict with Go to Symbol in Workspace

```json
    {
        "key": "ctrl+t",
        "command": "openInTerminal",
        "when": "!editorFocus"
    }
```

Reply:
Sure, but more specific like `filesExplorerFocus` would be better (I think that's the right one)